dockstore-webservice/src/main/java/io/dockstore/webservice/core/database/PartnerMetrics.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.dockstore.webservice.core.database;
+
+import io.dockstore.common.Partner;
+
+public record PartnerMetrics(long entryId, Partner partner) {
```

Comment:
If this isn't used by anything other than `Entry` and `EntryDAO`, it seems more organized to declare it nested inside Entry